cmdstanpy/cmds.py

```diff
@@ -54,7 +54,7 @@ def compile_model(
         if include_paths is not None:
             if any([not os.path.exists(d) for d in include_paths]):
                 raise Exception('no such include path {}'.format(d))
-            cmd = cmd.append('--include_paths=' + ','.join(include_paths))
+            cmd += ['--include_paths=' + ','.join(include_paths)]
```

Comment:
I think previous was better. Just remove `cmd = ` part, because `.append` does this inplace.

Reply:
Thanks!